packages/@aws-cdk/aws-cloudwatch/test/integ.alarm-and-dashboard.ts

```diff
@@ -53,6 +53,13 @@ dashboard.addWidgets(new cloudwatch.GraphWidget({
   title: 'More messages in queue with alarm annotation',
   left: [numberOfMessagesVisibleMetric],
   leftAnnotations: [alarm.toAnnotation()],
+  verticalAnnotations: [
```

Comment:
I'd prefer to see a new test that includes these annotations instead of editing an existing test.

Reply:
may I ask why? that would involve a whole lot of boilerplate setting up a new test, this was trivial to add

Reply:
Because when we pile too much functionality into one test, it's harder to decipher if we're getting the output we're looking for and harder to troubleshoot when a test fails.

Reply:
Also, was this test run and deployed? If so, I'd expect to see changes to more than these two files. It appears that this was run with `--dry-run` or that it was manually edited.